apps/examples/testcase/ta_tc/task_manager/itc/itc_taskmanager_main.c

```diff
@@ -247,108 +328,160 @@ void cleanup_task5(void)
 	unregister_task5_flag = true;
 }
 
-void itc_taskmanager_task3(void)
+void taskmanager_task3(void)
 {
 	int ret;
 	int tm_sample_handle4;
 	tm_appinfo_list_t *sample_list_info;
 
 	sample_list_info = ((tm_appinfo_list_t *)task_manager_getinfo_with_name("task5", TM_RESPONSE_WAIT_INF));
-	TC_ASSERT_NEQ("task_manager_getinfo_with_name", sample_list_info, NULL);
+	if (sample_list_info == NULL) {
+		printf("ERROR: failed task_manager_getinfo_with_name in taskmanager_task3\n");
+	}
 
 	tm_sample_handle4 = sample_list_info->task.handle;
 	task_manager_clean_infolist(&sample_list_info);
-	TC_ASSERT_NEQ("task_manager_clean_infolist", (tm_appinfo_list_t *)&sample_list_info, NULL);
+	if ((tm_appinfo_list_t *)&sample_list_info == NULL) {
```

Comment:
Is this condition correct?
after clean_infolist, it will be NULL

Reply:
fixed it